pkg/convert/xliff2_test.go

```diff
@@ -0,0 +1,197 @@
+package convert
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/text/language"
+)
+
+func TestFromXliff2(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name    string
+		wantErr error
+		data    []byte
+		want    model.Messages
+	}{
+		{
+			name: "All OK",
+			data: []byte(`<?xml version="1.0" encoding="UTF-8"?>
+<xliff version="2.0" xmlns="urn:oasis:names:tc:xliff:document:2.0" srcLang="en" trgLang="fr">
+  <file id="ngi18n" original="ng.template">
+    <unit id="common.welcome">
+      <notes>
+        <note category="location">src/app/app.component.html:16</note>
+      </notes>
+      <segment>
+        <source>Welcome!</source>
+        <target>Bienvenue!</target>
+      </segment>
+    </unit>
+    <unit id="common.app.title">
+      <notes>
+        <note category="location">src/app/app.component.html:4</note>
+        <note category="description">App title</note>
+      </notes>
+      <segment>
+        <source>Diary</source>
+        <target>Agenda</target>
+      </segment>
+    </unit>
+  </file>
+</xliff>`),
+			want: model.Messages{
```

Comment:
Maybe it's better to call it expected?
expected: model.Messages...

Reply:
Maybe you are right, but I guess it should be in a separate PR, as if I change I should change in all files to remain consistency